contiki/platform/linux/parse_config.c

```diff
@@ -421,8 +421,20 @@ void ConfigInit()
     } else {
       cfg.is_max_lr_powerlevel_set = 0;
     }
+
+    cfg.single_classic_temp_association = atoi(
+      config_get_val("single_classic_temp_association", "0"));
```

Comment:
Would it be preferable to align naming in conf file, I  am proposing:
```
ZipAssociationLimit=0,1
```

Reply:
OK with the use of `ZipAssociationLimit` in the configuration file.

Reply:
Would you like me to update your PR to squash the name ? Let me issue a PR to your fork

Reply:
https://github.com/lmolina/zipgateway/pull/1